File: src/main/java/guess/Game.java
package guess;

import java.util.Scanner;

public class Game {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int number = 0;
        while (number != -1) {
            System.out.print("Please enter a number: ");
            number = scanner.nextInt();
            System.out.println(number);
        }

        int i = 0;
        do {
            System.out.println(i);
            i++;
        } while ( i<3 );
    }
}

File: src/main/kotlin/com/courtney/Game.kt
package com.courtney

import java.util.*
import kotlin.random.Random

fun main() {
    var secret = Random.nextInt(10)+1
    println(secret)
    var number = 0
    while (number != secret) {
        print("Please enter a number: ")
        number = Scanner(System.`in`).nextInt()
        if (number > secret) {
            println("Lower")
        } else if (number < secret) {
            println("Higher")
        } else {
            println("Great! The number is: " + number)
        }
    }

}